src/main/scala/uk/gov/nationalarchives/tdr/api/service/FinalTransferConfirmationService.scala

```diff
@@ -51,3 +73,11 @@ object FinalTransferConfirmationService {
     LegalOwnershipTransferConfirmed
   )
 }
+
+object FinalJudgmentTransferConfirmationService {
```

Comment:
You don't need a second object here. It does work but the existing `FinalTransferConfirmationService` object on line 67 is the companion object for the class so you can add in `LegalCustodyTransferConfirmed` and `finalJudgmentTransferConfirmationProperties` into the existing object.